src/teleop/gui/README.md

```diff
@@ -0,0 +1,94 @@
+# GUI LCM Documentation 
+
+### Base Station GUI LCMs
+
+#### All GUIs:
+
+DebugMessage
+"/debug_message"
+File: DebugMessage.lcm
+Subscritions: SATask.vue, AutonTask.vue, RATask.vue
+Publishers: N/A
+Usage: Sends an error message
+
+Drive Velocity
+"/drive_vel_data"
+File: DriveVelData.lcm
+Subscriptions: DriveVelDataH.vue, DriveVelDataV.vue
+Publishers: N/A
+Usage: LCM data displayed in table on GUIs
+
+Encoder
+"/encoder"
+File: Encoder.lcm
+Subscriptions: SATask.vue, AutonTask.vue, RATask.vue
+Publishers: N/A
+Usage: 
+
+NavStatus
+"/nav_status"
+File: NavStatus.lcm
+Subscritions: SATask.vue, AutonTask.vue, RATask.vue
+Publishers: N/A
+Usage: 
+
+Odometry
+"/odometry"
+File: OdometryReading.lcm
+Subscritions: SATask.vue, AutonTask.vue, RATask.vue
+Publishers: N/A
+Usage: Displays odometry data on GUIs
+
+
+#### Science GUI:
+MosfetCmd
+"/mosfet_cmd"
```

Comment:
as a quick heads up, we were probably thinking about moving away from using the MosfetCmd msg. probably wait until u have the official icd with esw but yeah just a head's up before u start migrating everything